src/timetableGen/Timetable.java

```diff
@@ -53,4 +55,16 @@ public boolean hasConflict(){
 		return false;
 	}
 	
-}
+	public void genAllCombinations(Course course1, Course course2){
```

Comment:
all combinations should pick 1 meeting set from every course, not just 2 courses. you may try recursion in making this kind of situation. if you really want to use a loop, please make use of a stack structure